README.md

```diff
@@ -1,93 +1,25 @@
-# sql-to-mongo
+# MoQL
 
-Converts M-SQL Queries to Mongo find statements or aggregation pipelines.
+![build status](https://github.com/synatic/sql-to-mongo/actions/workflows/ci-build.yml/badge.svg)
 
-What is M-SQL?
+MoQL Converts SQL statements to Mongo find statements or aggregation pipelines. MoQL supports mySQL and Postgres Syntax, and generates Mongo 3.6 or greater compatible queries.
 
-M-SQL is a specific way to use MySQL style queries tailored to MongoDB functions and a multilevel document paradigm.
-
-## Notes
-
--   Supports Mongo 3.6 or greater
--   Follows mySQL Syntax
--   As with MongoDB column names and strings are case-sensitive.
-
-### Currently Unsupported SQL Statements
-
--   Over
--   CTE's
--   Pivot
--   Union
+For full docs and a playground to try MoQL out, vist [https://moql.synatic.dev/](https://moql.synatic.dev/)
 
 ## Installation
 
+Install MoQL using the [npm install command](https://docs.npmjs.com/downloading-and-installing-packages-locally):
+
 ```bash
 npm i @synatic/sql-to-mongo --save
```

Comment:
```suggestion
npm i @synatic/sql-to-mongo
```
You don't need `--save` these days, npm always saves to the package.json by default now